pkg/report/writer.go

```diff
@@ -204,6 +211,12 @@ func NewTemplateWriter(output io.Writer, outputTemplate string) (*TemplateWriter
 		"getCurrentTime": func() string {
 			return Now().UTC().Format(time.RFC3339Nano)
 		},
+		"filterForCVE": func(input string) string {
+			if !strings.Contains(strings.ToLower(input), "cve") {
+				return ""
+			}
+			return utils.GenerateAVDLink(input)
```

Comment:
Also, it may not be necessary. If there is an AVD link in `References`, we'll extract it, but if we don't find it, we can return the first element of `References`.